CMakeLists.txt

```diff
@@ -67,8 +67,15 @@ include(cmake/SetupPackages.cmake)
 include(cmake/SetupRajaConfig.cmake)
 
 # Configure a header file with all the variables we found.
+get_directory_property( RAJA_DEFINITIONS_INITIAL DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS )
+set(RAJA_DEFINITIONS "")
+foreach(definition ${RAJA_DEFINITIONS_INITIAL})
+  set(RAJA_DEFINITIONS "${RAJA_DEFINITIONS} -D${definition}")
+endforeach()
```

Comment:
That's just for the timers in LULESH right? We ought to rename it to LULESH_USE_X and then get rid of this junk?


________________________________
From: David Poliakoff
Sent: Friday, June 10, 2016 2:41:26 PM
To: LLNL/RAJA
Cc: Beckingsale, David Alexander; Comment
Subject: Re: [LLNL/RAJA] Added RAJA packaging capabilities (#51)


In CMakeLists.txt<https://github.com/LLNL/RAJA/pull/51#discussion_r66684999>:

> @@ -67,8 +67,15 @@ include(cmake/SetupPackages.cmake)
>  include(cmake/SetupRajaConfig.cmake)
>
>  # Configure a header file with all the variables we found.
> +get_directory_property( RAJA_DEFINITIONS_INITIAL DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS )
> +set(RAJA_DEFINITIONS "")
> +foreach(definition ${RAJA_DEFINITIONS_INITIAL})
> +  set(RAJA_DEFINITIONS "${RAJA_DEFINITIONS} -D${definition}")
> +endforeach()


Well, it also needs -DRAJA_USE_GETTIME, which is added through add_definitions

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/LLNL/RAJA/pull/51/files/b8674a47a4fecac04215882565afe5b0532d4b6b#r66684999>, or mute the thread<https://github.com/notifications/unsubscribe/AAUakwQQNglpx3Y3hekqqaQ43F1S_Mfxks5qKdoGgaJpZM4IzV4_>.